grabcut/grabcut.py

```diff
@@ -320,8 +320,8 @@ def view(self):
         dmap = hv.DynamicMap(self.extract_foreground)
         dmap = hv.util.Dynamic(dmap, operation=self._filter_contours)
         dmap = hv.util.Dynamic(dmap, operation=self._simplify_contours)
-        return (regrid(self.image).options(**options) * self.bg_paths * self.fg_paths +
```

Comment:
Why was the regrid add here? It kept making the shapes mismatched and took me forever to find it :(

Reply:
Presumably to make it possible to display *really* large images without causing issues in the browser? That would be my initial guess at least.